File: API/Entities/MenuItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class MenuItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Restuarant { get; set; }
        public float Price { get; set; }
        public string ImgUrl { get; set; }
        public string PublicId { get; set; }
        public string prepTime { get; set; }
        public string Category { get; set; }
    }
}

File: API/client/src/app/_models/menuItem.ts
export interface MenuItem {
  id: number;
  name: string;
  description: string;
  category: string;
  price: string;
  imgUrl: string;
  prepTime: string;
  minimumPrice: number;
  rate: number;
  availability: boolean;
  publicId: string;
  subCategory: string;
  weight:string;
  flavours: string[];
  meatTemperatures: string[];
  sauces: string[];
  selectedFlavour: string;
  selectedMeatTemperature: string;
  selectedSauces: string[];
}
